pkgs/epnix/support/ads/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  epnixLib,
+  epnix,
+  mkEpicsPackage,
+  fetchFromGitLab,
+  local_config_site ? {},
+  local_release ? {},
+}:
+mkEpicsPackage rec {
+  pname = "ads";
```

Comment:
Is there any reason to call this package `ads` instead of `twincat-ads`? I figured that since the repo is named `epics-twincat-ads`, it might help with the ambiguity between this module and Cosylab's module

Reply:
agree